tony-core/src/test/java/com/linkedin/tony/TestHistoryFileUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.tony;
```

Comment:
Put the test for HistoryFileUtils in the same package as HistoryFileUtils, i.e. `com.linkedin.tony.util`.

Reply:
By this logic, I'm assuming all the other utils test (TestUtils, TestHdfsUtils, TestParserUtils, etc.) should also be moved to `test.com.linkedin.tony.util`?

Reply:
Package name should not be prefixed `test.` (I think you're probably referring to `src/test/java/com...`), but yes, that's the general pattern.